infrastructure/oss/issue.go

```diff
@@ -45,8 +46,23 @@ func toIssue(
 	ep error_reporting.ErrorReporter,
 ) snyk.Issue {
 	// this needs to be first so that the lesson from Snyk Learn is added
-	codeActions := issue.AddCodeActions(learnService, ep)
-
+	codeActions := issue.AddCodeActions(learnService, ep, affectedFilePath, issueRange)
+
+	var codelensCommands []snyk.CommandData
+	for _, codeAction := range codeActions {
+		fmt.Println(codeAction.Title)
```

Comment:
Can't this disturb LS stdio comms, if we output something to stdout? Is this a debug left-over?

Reply:
Yes it is a debug left-over, thanks for noticing it!